src/main/scala/io/gatling/sbt/settings/BaseSettings.scala

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2011-2022 GatlingCorp (https://gatling.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.gatling.sbt.settings
+
+import io.gatling.sbt.settings.gatling._
+import io.gatling.sbt.utils.PropertyUtils.DefaultJvmArgs
+
+import _root_.io.gatling.sbt.GatlingPlugin.gatlingTestFramework
+
+import sbt._
+import sbt.Keys._
+import sbt.Tests.{ Argument, Group }
+
+object BaseSettings {
+
+  private def overrideDefaultJavaOptions(javaOptions: String*): Seq[String] =
```

Comment:
How else we could configure java options? As this is private anymore

Reply:
We're going to revert. Thanks for reporting.

Reply:
We've released 4.1.1 with the revert.